### ISSUE TITLE
ModuleNotFoundError: No module named 'ipware.ip2'  -- error in django-axes==5.0.7, Django==2.2, Python==3.8

### ISSUE THREAD

**Issue by mnislam01** [jazzband/django-axes#601]
`django-ipware` has no module `ipware.ip2` in recent version. Please fix this ASAP.

In the `helpers.py` file import `ipware.ip2` is breaking for new version of django-ipware.

```
django_1         | Traceback (most recent call last):
django_1         |   File "manage.py", line 30, in <module>
django_1         |     execute_from_command_line(sys.argv)
django_1         |   File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
django_1         |     utility.execute()
django_1         |   File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 357, in execute
django_1         |     django.setup()
django_1         |   File "/usr/local/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
django_1         |     apps.populate(settings.INSTALLED_APPS)
django_1         |   File "/usr/local/lib/python3.7/site-packages/django/apps/registry.py", line 122, in populate
django_1         |     app_config.ready()
django_1         |   File "/usr/local/lib/python3.7/site-packages/axes/apps.py", line 47, in ready
django_1         |     from axes import checks, signals  # noqa
django_1         |   File "/usr/local/lib/python3.7/site-packages/axes/signals.py", line 11, in <module>
django_1         |     from axes.handlers.proxy import AxesProxyHandler
django_1         |   File "/usr/local/lib/python3.7/site-packages/axes/handlers/proxy.py", line 7, in <module>
django_1         |     from axes.handlers.base import AxesHandler
django_1         |   File "/usr/local/lib/python3.7/site-packages/axes/handlers/base.py", line 2, in <module>
django_1         |     from axes.helpers import (
django_1         |   File "/usr/local/lib/python3.7/site-packages/axes/helpers.py", line 11, in <module>
django_1         |     import ipware.ip2
django_1         | ModuleNotFoundError: No module named 'ipware.ip2'

```

**Comment by Huertix** [comment]
Same in here

**Comment by Kingjmk** [comment]
Same here, please update django-ipware dependency

**Comment by georges-g** [comment]
I confirm the issue. Quickfix: change ipware.ip2 to ipware.ip in the helpers file (twice in the file).

**Comment by cheesysam** [comment]
@Huertix 

> Same in here
> 
> 404... https://github.com/un33k/django-ipware/blob/master/ipware/ip2.py
> 
> I hope you can fix this soon guys... our prod envs can not be deployed.. :(

Locking django-ipware in your own requirements file should allow you to build for now until this is sorted.

`django-ipware==2.1.0`

**Comment by Huertix** [comment]
Highly appreciate it @cheesysam

**Comment by aleksihakli** [comment]
Fixed in 5.3.5 where ipware has been locked to suitable version range >=2,<3 for now.
